Dockerfile

```diff
@@ -25,7 +25,16 @@ FROM ${GO_BORING} AS builder-amd64
 FROM builder AS builder-s390x
 FROM builder-${ARCH} AS calico-node-builder
 ARG ARCH
+ARG GOLANG_VERSION=1.17.5
+ARG GOBORING_BUILD=7
```

Comment:
Add these variables on top so that we can quickly see what versions we are using

Reply:
Even if you add them on top, we should still write here `ARG GOLANG_VERSION` and `ARG GOBORING_BUILD` for the image to use them